doc/DISTRIB/ATS-Postiats/RELEASE/ats2-postiats-0.3.10-release.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>ATS2-0.3.10</title>
+</head><!--head-->
+
+<body>
+
+<H2>
+ats2-postiats-0.3.10:
+</H2>
+
+<pre>
+This is the 47th release of ATS2, the successor of the
```

Comment:
Something does not look quite right here. Is it the 48th or the 47th release?

Reply:
I checked the record in the following file:

https://github.com/githwxi/ATS-Postiats/blob/master/CHANGES-ats2

This one (ATS2-0.3.10) is the 47th.